sdk/include/opentelemetry/sdk/trace/span_data.h

```diff
@@ -257,7 +257,7 @@ class SpanData final : public Recordable
                 opentelemetry::common::SystemTimestamp timestamp =
                     opentelemetry::common::SystemTimestamp(std::chrono::system_clock::now()),
                 const opentelemetry::common::KeyValueIterable &attributes =
-                    opentelemetry::common::KeyValueIterableView<std::map<std::string, int>>(
+                    opentelemetry::common::KeyValueIterableView<std::map<std::string, int32_t>>(
                         {})) noexcept override
   {
     SpanDataEvent event(std::string(name), timestamp, attributes);
```

Comment:
This is a virtual method, which `override` the parent.

A better fix is to remove the default value entirely (remove `= ...`), as this is taken care of in class Recordable.

Could you check if it resolves the issue for your platform ?

Reply:
Fixed, pushed. Yes this helps on my platform.
But I had to remove 2 default params, since second can't be not-defaulted, while first has default value.